src/cirq_iqm/iqm_sampler.py

```diff
@@ -149,12 +119,15 @@ def run_sweep(
     def _send_circuits(
             self,
             circuits: list[cirq.Circuit],
-            repetitions: int = 1,
+            qubit_mapping: dict[str, str],
```

Comment:
```suggestion
            qubit_mapping: Optional[dict[str, str]],
```
It can still be None at this point, right?

Reply:
yes